.gitignore

```diff
@@ -3,7 +3,6 @@
 .bundle
 .config
 .yardoc
-Gemfile.lock
```

Comment:
I believe this is intentional. The gemspec should have sufficient information, no?

Reply:
E.g., if it fails tests as you describe in one place and not another, that means some version of the gem we are allowing is causing something to break. That would likely mean it is breaking for some user. That user won't be using our lock, they'll have their own.

Reply:
So the intention is for users to manage the dependencies rather than having to do so in this repo?

> The gemspec should have sufficient information, no?

The gemspec doesn't specify versions for the most part, but perhaps just major versions of gems might be a start if putting a `Gemfile.lock` isn't desirable. 🤔